docs/build.gradle

```diff
@@ -87,12 +85,6 @@ antora {
     ]
 }
 
-tasks.named('generateAntoraYml') {
-    asciidocAttributes = project.provider {
-        return ['micrometer-version': project.version.toString()]
```

Comment:
The `micrometer-version` property is used in an `.adoc` file, I think we need the `spring-antora-plugin` and the `generateAntoraYml` task to make it work.

The `generateAntoraResources` task can be removed (the `antora` task and the `antora.yml` need to be modified), though I would keep it as is even if it just "delegates" to `generateAntoraYml` in case we will have other antora-specific resources to generate.

Reply:
Reverted the change.